src/components/forms/label/Label.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from "clsx";
+import React, { FC } from "react";
+
+export interface LabelProps {
+  id: string;
+  children: string;
+  className?: string;
+}
+
+const Label: FC<LabelProps> = ({ id, className, children }) => (
```

Comment:
:+1: for adding this to be used with the `Textarea` component.

However I think it is a problem that we do not use it within the current `TextInput` component which served a similar purpose.

I suggest that you either use the new `Label` component both places or drop it for now and fall back to a simple `<label>` element.